PacketDotNet/IPv6ExtensionHeader.cs

```diff
@@ -0,0 +1,70 @@
+﻿using PacketDotNet.MiscUtil.Conversion;
+using PacketDotNet.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+/*
+This file is part of PacketDotNet
+
+PacketDotNet is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+PacketDotNet is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with PacketDotNet.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ * Copyright 2018 Steven Haufe<haufes@hotmail.com>
+ */
+
+namespace PacketDotNet
+{
+    [Serializable]
+    internal class IPv6ExtensionHeader
+    {
+        static public List<IPProtocolType> extensionHeaderTypes = new List<IPProtocolType>{ IPProtocolType.HOPOPTS,
```

Comment:
reorder to public static, should probably be renamed to something like KnownTypes. Can also be a hashset to improve lookups. I think the two reserved types (253 and 254) could be added as well.

Reply:
(following the later comments, it might better be defined in another file)